x/dispensation/keeper/querier.go

```diff
@@ -54,8 +57,9 @@ func queryDistributionRecordsForRecipient(ctx sdk.Context, req abci.RequestQuery
 	if err != nil {
 		return nil, sdkerrors.Wrap(sdkerrors.ErrJSONUnmarshal, err.Error())
 	}
-	records := keeper.GetRecordsForRecipient(ctx, params.Address)
-	res, err := codec.MarshalJSONIndent(keeper.cdc, records)
+	addr, _ := sdk.AccAddressFromBech32(params.Address)
```

Comment:
Is it safe ignore the error check here ?

Reply:
Updated.